plugins/nid-team/scripts/pr-dashboard/sync-dashboard.sh

```
@@ -716,3 +715,28 @@ for item in data['items']:
+# --- Post project status update ---
+echo ""
+echo "--- Posting project status update ---"
+
+TOTAL_ADDED=$((SHARED_ADDED + BUGPR_ADDED + DOCS_ADDED))
+STATUS_BODY="**Dashboard synced** ($(date '+%b %-d, %Y %H:%M %Z'))"
+if [ "$TOTAL_ADDED" -gt 0 ]; then
+    STATUS_BODY="$STATUS_BODY\n- $TOTAL_ADDED PRs added ($SHARED_ADDED shared, $BUGPR_ADDED bug, $DOCS_ADDED docs)"
+fi
+STATUS_BODY="$STATUS_BODY\n- $ITEM_COUNT total items on board"
+
+ESCAPED_BODY=$(printf "$STATUS_BODY" | python3 -c "import sys,json; print(json.dumps(sys.stdin.read()))")
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)